fixtures/martin/quantum-wash.json

```diff
@@ -0,0 +1,177 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Quantum Wash",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Ryan Brady"],
+    "createDate": "2023-12-19",
+    "lastModifyDate": "2023-12-19"
+  },
+  "links": {
+    "manual": [
+      "https://www.martin.com/en-US/products/mac-quantum-wash"
+    ]
+  },
+  "rdm": {
+    "modelId": 34654457,
+    "softwareVersion": "V1"
+  },
+  "availableChannels": {
+    "Shutter / Strobe": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "ShutterStrobe",
+        "shutterEffect": "Open"
+      }
+    },
+    "Dimmer Fade": {
+      "fineChannelAliases": ["Dimmer Fade fine"],
+      "defaultValue": "0%",
+      "capabilities": [
+        {
+          "dmxRange": [0, 255],
+          "type": "Intensity"
+        },
+        {
+          "dmxRange": [256, 65535],
+          "type": "Intensity"
+        }
+      ]
+    },
+    "Red": {
+      "fineChannelAliases": ["Red fine"],
+      "defaultValue": "0%",
+      "capabilities": [
+        {
+          "dmxRange": [0, 255],
+          "type": "ColorIntensity",
+          "color": "Red"
+        },
+        {
+          "dmxRange": [256, 65535],
+          "type": "ColorIntensity",
+          "color": "Red"
+        }
+      ]
+    },
+    "Green": {
+      "fineChannelAliases": ["Green fine"],
+      "defaultValue": "0%",
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Dimmer": {
+      "fineChannelAliases": ["Dimmer fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Red 2": {
+      "name": "Red",
+      "fineChannelAliases": ["Red 2 fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Blue": {
+      "fineChannelAliases": ["Blue fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "CTC": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "0%",
+          "colorTemperatureEnd": "0%"
+        },
+        {
+          "dmxRange": [11, 170],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "2000K",
+          "colorTemperatureEnd": "10000K"
```

Comment:
The manual is a little funny here (overlapping 171), but I think since it says "50 K steps", the range should be [11, 171] here, and [172, 255] for the constant value below.